### PR TITLE
fix(DB/Azshara): adjusting y and z axes for node

### PR DESCRIPTION
**Pull request by damnablebear**
Adjusted position Y and position Z to make node spawn a bit more present and less above-ground than it was before. Sticks out of the ground an appropriate amount like other nodes now.

## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/1247

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by damnablebear** [comment]
This screenshot shows the object's new positioning:
image

**Comment by heyitsbench** [comment]
You'll want to move the SQL file into [data/sql/updates/pending_db_world](https://github.com/damnablebear/azerothcore-wotlk/tree/truesilver-node-bump/data/sql/updates/pending_db_world).

**Comment by Gultask** [comment]
Since most of these spawns aren't already sniffed and will be reworked at a later date, this type of fix can be accepted.
Just one question:

Is this node part of a pool? You can find out if you go to the location and use the command `.gob near`, it'll show you if there are any possible spawns nearby, in case any more need adjusting.
You can also find out by searching for the guid in the pool_gameobject table. If you have questions I can try to explain how it works as well.

File: truesilver-azshara-spawn-guid-9329-update.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `gameobject` SET `position_y` = -5124.500500, `position_z` = 87.242325 WHERE `guid` = 9329;
```
Comment (Gultask) [inline truesilver-azshara-spawn-guid-9329-update.sql:1]: What Bench said, but also you need, when UPDATEing the gameobject or creature table, also specify the id of the gameobject in question.
```sql
WHERE `guid` = X AND `id` = Y;
```

**Comment by Tereneckla** [comment]
New lines have to be linux style

**Comment by damnablebear** [comment]
@Gultask to answer your earlier question about pools, I don't believe it is. This is what I see when I run that command.

image

**Review by Maria-sequel — APPROVED** [review]

**Review by elthehablo — APPROVED** [review]

**Review by Gultask — APPROVED** [review]

**Pull request merged**